lib/transport.rb

```diff
@@ -0,0 +1,24 @@
+class Transport
+  include Comparable
+
+  attr_accessor :max_weight, :speed, :available
+  attr_reader :delivery_speed_by_weight
+
+  def <=>(other)
+    delivery_speed_by_weight <=> other.delivery_speed_by_weight
+  end
+
+  def initialize(max_weight, speed, available)
+    @max_weight = max_weight
+    @speed = speed
```

Comment:
lets to speed.to_f in inizialize method instead of using it twice (delivery_time and delivery_speed_by_weight methods)